### PR TITLE
docs: links added for each builder schema

### LINKED ISSUE #43542: @angular-devkit/build-angular package link points to 404
### Description

The link in the section [Configuring Builder targets](https://angular.io/guide/workspace-config#default-architect-builders-and-targets) of the Angular workspace configuration page
redirects to :
https://github.com/angular/angular-cli/blob/master/packages/angular/cli/lib/config/schema.json
which results in a 404 github page

I think (not 100% sure) that the url should be:
https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/builders.json

Screenshot at 2021-09-22 22-43-48

### PR DESCRIPTION
**Pull request by enisfr**
closes #43542

Signed-off-by: enisfr

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/workspace-config.md [hunk 1]
```diff
@@ -96,16 +96,16 @@ See [Angular CLI Builders](guide/cli-builder).
...
-*   browser
-*   dev-server
-*   extract-i18n
-*   karma
-*   server
-*   tslint
+The JSON schemas that define the options and defaults for each of these default builders are collected in the [`@angular-devkit/build-angular`](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/builders.json) package.
+The schemas configure options for the following builders.d
+
+* [app-shell](https://github.com/angular/angular-cli/tree/master/packages/angular_devkit/build_angular/src/builders/app-shell)
```
Comment (alan-agius4) [inline aio/content/guide/workspace-config.md:102]: Can you please update all links to point to the schema file? 

Example
```suggestion
* [app-shell](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/builders/app-shell/schema.json)
```

Comment (enisfr) [reply aio/content/guide/workspace-config.md]: updated links pointing to the schema files

File: aio/content/guide/workspace-config.md [hunk 2]
```diff
@@ -96,16 +96,16 @@ See [Angular CLI Builders](guide/cli-builder).
...
+The JSON schemas that define the options and defaults for each of these default builders are collected in the [`@angular-devkit/build-angular`](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/builders.json) package.
+The schemas configure options for the following builders.d
+
+* [app-shell](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/builders/app-shell/schema.json)
+* [browser](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/builders/browser/schema.json)
+* [dev-server](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/builders/dev-server/schema.json)
+* [extract-i18n](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/builders/extract-i18n/schema.json)
+* [karma](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/builders/karma/schema.json)
+* [server](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/builders/server/schema.json)
+* tslint
```
Comment (alan-agius4) [inline aio/content/guide/workspace-config.md:108]: This can be removed since tslint builder is no longer available

**Review by alan-agius4 — APPROVED** [review]
LGTM, can you please squash your commits?

File: aio/content/guide/workspace-config.md (hunk 2 above)
Comment (enisfr) [reply aio/content/guide/workspace-config.md:108]: I will remove tslint and also squash the commit, is that ok for this pr?

Comment (alan-agius4) [reply aio/content/guide/workspace-config.md:108]: Yes

**Comment by alan-agius4** [comment]
Thanks for your contribution.

**Pull request closed without merging**